roles/container-engine/containerd-common/defaults/main.yml

```diff
@@ -3,15 +3,3 @@
 # manager controlled installs to direct download ones.
 containerd_package: 'containerd.io'
 yum_repo_dir: /etc/yum.repos.d
```

Comment:
The `yum_repo_dir` here appears to be duplicated with the one in this file: https://github.com/kubernetes-sigs/kubespray/blob/master/roles/system_packages/defaults/main.yml

Should we consider removing this variable as well?